README.md

```diff
@@ -6,6 +6,15 @@ rust-rocksdb
 
 Feedback and pull requests welcome!  If a particular feature of RocksDB is important to you, please let me know by opening an issue, and I'll prioritize it.
 
-This binding is statically linked with a specific version of RocksDB. If you want to build it yourself, make sure you've also cloned the RocksDB and Snappy submodules:
+This binding is statically linked with a specific version of RocksDB. If you want to build it yourself, make sure you've also cloned the RocksDB and compression submodules:
 
     git submodule update --init --recursive
+
+## Compression Support
+By default, support for the [Snappy](https://github.com/google/snappy), [LZ4](https://github.com/lz4/lz4), [Zstd](https://github.com/facebook/zstd), [Zlib](https://zlib.net), and [Bzip2](http://www.bzip.org) compression is enabled through crate features.  If support for all of these compression algorithms is not needed, default features can be disabled and specific compression algorithms can be enabled. For example, to enable only LZ4 compression support, make these changes to your Cargo.toml:
```

Comment:
Well, this is not true, as far as I can see - default features includes `static` feature only.

Reply:
I also recommend you to use 80-chars long lines with explicit newlines

Reply:
> 
> 
> Well, this is not true, as far as I can see - default features includes `static` feature only.

The [default is set](https://github.com/rust-rocksdb/rust-rocksdb/pull/209/files#diff-80398c5faae3c069e4e6aa2ed11b28c0R17) at the main crate, not the `-sys` crate.

Reply:
> 
> 
> I also recommend you to use 80-chars long lines with explicit newlines

In the README, or somewhere else?

Reply:
Oh, my bad, I see now. 
> In the README, or somewhere else?

I prefer to see them everywhere, actually. The main reason is that it's much easier to review and modifying some words in the middle of the sentence usually doesn't produce huge diffs.

Reply:
Modified the README to wrap the lines.